lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -1592,8 +1592,6 @@ void ByteCodeGenerator::EmitScopeObjectInit(FuncInfo *funcInfo)
     Js::PropertyIdArray *propIds = funcInfo->GetParsedFunctionBody()->AllocatePropertyIdArrayForFormals(extraAlloc, slotCount, Js::ActivationObjectEx::ExtraSlotCount());
 
     ParseNodeFnc *pnodeFnc = funcInfo->root;
-    ParseNode *pnode;
-    Symbol *sym;
```

Comment:
These declarations were moved closer to their point-of-use in order to avoid variable name shadowing.